mmv1/templates/terraform/custom_flatten/cloudfunctions2_function_source_bucket.go.erb

```diff
@@ -14,5 +14,8 @@
 # limitations under the License.
 -%>
 func flatten<%= prefix -%><%= titlelize_property(property) -%>(v interface{}, d *schema.ResourceData, config *Config) interface{} {
-return d.Get("build_config.0.source.0.storage_source.0.bucket")
+    if _, ok := d.GetOk("build_config.0.source.0.storage_source.0.bucket"); ok {
+        return d.Get("build_config.0.source.0.storage_source.0.bucket")
+    }
```

Comment:
```suggestion
    if bVal, ok := d.GetOk("build_config.0.source.0.storage_source.0.bucket"); ok {
        return bVal
    }
```

`GetOk` also does a `Get`, you can use that value instead of re-retrieving it.

Reply:
Yeah, we can do that.